apollo-router/src/plugins/telemetry/config_new/router/selectors.rs

```diff
@@ -216,7 +226,7 @@ impl Selector for RouterSelector {
             } => get_baggage(baggage).or_else(|| default.maybe_to_otel_value()),
             RouterSelector::Static(val) => Some(val.clone().into()),
             RouterSelector::StaticField { r#static } => Some(r#static.clone().into()),
-            RouterSelector::ResponseBody { response_body } if *response_body => {
+            RouterSelector::ResponseBody { response_body, truncate: _truncate } if *response_body => {
```

Comment:
Since we don't use `truncate` here, this should be fine:
```suggestion
            RouterSelector::ResponseBody { response_body, .. } if *response_body => {
```